pyproject.toml

```diff
@@ -23,7 +23,10 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 requires-python = ">=3.8"
-dependencies = []
+dependencies = [
+    "pydantic",
```

Comment:
Which versions of pydantic do we support?

Reply:
Probably only the latest major, to be determined. I am hoping to make the tests comprehensive enough that we will support the lowest version where tests pass.

Reply:
We agreed to try and support the lowest version possible within the pydantic 1.0+ range that does not require major compatibility code.